packages/marketplace/contracts/libraries/LibAsset.sol

```diff
@@ -83,6 +83,8 @@ library LibAsset {
     /// @param rightClass The asset class type of the right side of the trade.
     /// @return FeeSide representing which side should bear the fee, if any.
     function getFeeSide(AssetClass leftClass, AssetClass rightClass) internal pure returns (FeeSide) {
+        require((leftClass == AssetClass.ERC20 || rightClass == AssetClass.ERC20), "exchange not allowed");
```

Comment:
This is doing too much, its essentially blocking ERC721 <> ERC721 trade and its out of scope for us.
We just want to make sure that if one side is BUNDLE the other is ERC20

Example:
```suggestion
     if (leftClass == AssetClass.BUNDLE || rightClass == AssetClass.BUNDLE) {
            require(
                ((leftClass == AssetClass.BUNDLE && rightClass == AssetClass.ERC20) ||
                    (rightClass == AssetClass.BUNDLE && leftClass == AssetClass.ERC20)),
                "exchange not allowed"
            );
        }
        }
```